index.bs

```diff
@@ -131,9 +131,26 @@ Device identification and classification (e.g. device type and class) based on a
 
 To reduce fingerprinting risk, reported value is rounded to single most significant bit, as opposed to reporting the exact value. In addition, an upper and lower bound is placed on the reported values.
 
+IANA considerations {#iana}
+===================
+This document defines the `Device-Memory` HTTP request header field, and register them in the permanent message header field registery ([[RFC3864]]).
+
+Device-Memory Header field {#iana-device-memory}
+---------------------------------
+Header field name: Device-Memory
```

Comment:
Should this use bikeshed's definition list syntax?

```
:Header field name
:: `Device-Memory`
... etc
```

Reply:
Sure!